generator/template/default/goss.yaml

```diff
@@ -0,0 +1,10 @@
+process:
```

Comment:
Should this file be optional? Might surprise many (like me) who dont know what goss is

Reply:
Yeah, good point.

Reply:
done.